mozetl/search/aggregates.py

```diff
@@ -238,10 +238,14 @@ def generate_rollups(submission_date, output_bucket, output_prefix,
     logger.info('Applying transformation function...')
     search_dashboard_data = transform_func(main_summary)
 
+    if orderBy:
+        ordered_data = search_dashboard_data.orderBy(*orderBy)
+    else:
+        ordered_data = search_dashboard_data
```

Comment:
I'm somewhat confused -- it looks like we're assigning to this ordered_data variable, but never using it?

Are you sure you don't want just:

```py
if orderBy:
  search_dashboard_data = search_dashboard_data.orderBy(*orderBy)
...
```

?

Reply:
Gah, my mistake. Fixed now.